### ISSUE TITLE
Model cant be found but loading model

### ISSUE THREAD

**Issue by BartSkol** [TheLastBen/fast-stable-diffusion#1502]
I deleted old sd folder, used new colab version of notebook, but still, when i try to generate anything it gives me just latent noise.
![00001-3172461088-test](https://user-images.githubusercontent.com/65338669/215222947-facaa3e3-bfd7-43d8-91b6-c7e03a65b917.png)
"Model loaded in 30.7s (calculate hash: 12.2s, load weights from disk: 6.7s, create model: 0.8s, apply weights to model: 3.6s, apply half(): 1.5s, load VAE: 5.1s, move model to device: 0.8s, load textual inversion embeddings: 0.1s).
Checkpoint model.ckpt [43c90409bd] not found; loading fallback model.ckpt [43c90409bd]"

**Comment by richvip93** [comment]
Same here! Just with 2.1 versions

**Comment by eliasprompt** [comment]
1.5 models do not charge either, nor do the VAE

**Comment by richvip93** [comment]
I am using my own model in 1.5 and it works fine, in case it helps

**Comment by eliasprompt** [comment]
no, they don't work, it seems that they load and images can be made but they are based on the basic 1.5 model, the error can be checked on the colab page as the colleague has commented.

**Comment by BartSkol** [comment]
So as for now, 2.1 is not working with this version of fast-stable-diffusion? Is there anything that I could do to make it work again?

**Comment by TheLastBen** [comment]
Open an issue in A1111 repo, the v2-768 is not automatically detected
